src/man/sssd-simple.5.xml

```diff
@@ -40,25 +40,31 @@
             apply:
             <itemizedlist>
                 <listitem>
-                    <para>If all lists are empty, access is granted</para>
+                    <para>
+                        It is not recommended to leave an option empty, it might
```

Comment:
@justin-stephenson tagging you

Reply:
??

Reply:
I removed that section and moved the paragraph about empty list here. wdyt?

Reply:
Looks good to me.